src/aliceVision/mesh/Texturing.cpp

```diff
@@ -481,11 +481,9 @@ void Texturing::generateTexturesSubSet(const mvsUtils::MultiViewParams& mp,
         }
         ALICEVISION_LOG_INFO("- camera " << camId + 1 << "/" << mp.ncams << " with contributions to " << cameraContributions.size() << " texture files:");
 
+        //Load camera image from cache
         imageCache.refreshData(camId);        
-
-        //conversion ImagesCache::Img -> Image
-        mvsUtils::ImagesCache::ImgPtr imgPtr = imageCache.getImg_sync(camId);
-        Image camImg(imgPtr->data, imgPtr->getWidth(), imgPtr->getHeight());
+        const Image& camImg = *(imageCache.getImg_sync(camId));
```

Comment:
`getImg_sync` returns an `ImgPtr` which is a `std::shared_ptr<Img>`.
If you remove the temporary variable and directly get the value from the shared_ptr, the shared_ptr ownership is removed.